SofaKernel/modules/SofaBaseMechanics/UniformMass.inl

```diff
@@ -366,10 +361,10 @@ void UniformMass<DataTypes, MassType>::handleTopologyChange()
     {
         indices.clear();
         for(size_t i=0; i<mstate->getSize(); i++)
-            indices.push_back((int)i);
+            indices.push_back(int(i));
```

Comment:
maybe indices should be a vector<unsigned int> instead of int, would avoid all the conversions